cmd/juju/model/dump.go

```diff
@@ -54,6 +55,7 @@ func (c *dumpCommand) Info() *cmd.Info {
 func (c *dumpCommand) SetFlags(f *gnuflag.FlagSet) {
 	c.ControllerCommandBase.SetFlags(f)
 	c.out.AddFlags(f, "yaml", output.DefaultFormatters)
+	f.BoolVar(&c.simplified, "simplified", false, "Dump a simplified partial model")
```

Comment:
Good that this defaults to false, but we should print a warning if true and only v2 api is available, lest the user expects they will get a simplified model and then they don't get one.